### PR TITLE
reaper: 6.25 -> 6.28

### PR DESCRIPTION
**Pull request by ilian**
###### Motivation for this change
Update Reaper to version 6.28.

Changelog: https://www.reaper.fm/whatsnew.txt

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by zakame** [comment]
Result of `nixpkgs-review pr 118061` run on Other x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>reaper</li>
  </ul>
</details>

On the build log, I see these warnings, should these be of any concern?

```
Cleaning up destination...rmdir: failed to remove '/nix/store/r3jmpn5dsklhi16c1y1nh5czix4m8433-reaper-6.26/opt/REAPER/InstallData/Effects/ix': No such file or directory
rmdir: failed to remove '/nix/store/r3jmpn5dsklhi16c1y1nh5czix4m8433-reaper-6.26/opt/REAPER/InstallData/Data/IX_scales': No such file or directory
rmdir: failed to remove '/nix/store/r3jmpn5dsklhi16c1y1nh5czix4m8433-reaper-6.26/opt/REAPER/InstallData/Data/IX_sequences': No such file or directory
 done
Copying files... done
Writing uninstall script to /nix/store/r3jmpn5dsklhi16c1y1nh5czix4m8433-reaper-6.26/opt/REAPER/uninstall-reaper.sh
Installing desktop integration for user..../install-reaper.sh: line 527: xdg-icon-resource: not found
./install-reaper.sh: line 528: xdg-icon-resource: not found
./install-reaper.sh: line 532: xdg-icon-resource: not found
./install-reaper.sh: line 532: xdg-icon-resource: not found
./install-reaper.sh: line 532: xdg-icon-resource: not found
./install-reaper.sh: line 532: xdg-icon-resource: not found
./install-reaper.sh: line 532: xdg-icon-resource: not found
./install-reaper.sh: line 532: xdg-icon-resource: not found
./install-reaper.sh: line 535: xdg-desktop-menu: not found
./install-reaper.sh: line 601: xdg-mime: not found
./install-reaper.sh: line 607: which: not found
./install-reaper.sh: line 611: which: not found
 done
```

**Review by zakame — APPROVED** [review]
LGTM, thanks! :tada:

**Review by evenbrenden — APPROVED** [review]

File: pkgs/applications/audio/reaper/default.nix [hunk 1]
```diff
@@ -28,6 +30,8 @@ stdenv.mkDerivation rec {
     alsaLib
     stdenv.cc.cc.lib # reaper and libSwell need libstdc++.so.6
     gtk3
+    xdg-utils # Required for desktop integration
+    which
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/reaper/default.nix:34]:
```suggestion
```

File: pkgs/applications/audio/reaper/default.nix [hunk 2]
```diff
@@ -8,18 +8,20 @@
...
-  version = "6.25";
+  version = "6.27";
 
   src = fetchurl {
     url = "https://www.reaper.fm/files/${lib.versions.major version}.x/reaper${builtins.replaceStrings ["."] [""] version}_linux_x86_64.tar.xz";
-    sha256 = "0i1idlr4ar28wvwcvwn9hqzb63kki1x1995cr87a9slxfa7zcshb";
+    hash = "sha256-6zHZK1ZYVSuCwZFWiBNJWhlk7Sau/paw+FMAaiVJHbg=";
   };
 
   nativeBuildInputs = [ autoPatchelfHook makeWrapper ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/reaper/default.nix:27]:
```suggestion
  nativeBuildInputs = [
    autoPatchelfHook
    makeWrapper
    xdg-utils # Required for desktop integration
    which
  ];
```

they are used in install-reaper.sh.

Comment (ilian) [reply pkgs/applications/audio/reaper/default.nix]: Fixed. Thanks.

**Comment by lukegb** [comment]
Thanks!

**Pull request merged**